docs/contributors/js-build-system.md

```diff
@@ -0,0 +1,55 @@
+# JS build system
+
+WooCommerce Blocks uses Webpack to build the files that will be consumed by browsers. There are several different Webpack configs in order to build files for different contexts of the plugin. They can all be found in [`webpack.config.js`](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/main/webpack.config.js#L148-L160), but this is a quick summary:
+
+-   `CoreConfig`: config for shared libraries like settings, blocks data or some HOCs and context.
+-   `MainConfig`: config that builds the JS files used by blocks in the editor and responsible for registering the blocks in Gutenberg.
```

Comment:
Typo - `the JS files used by blocks in the editor and is responsible for registering the blocks in Gutenberg`.

Reply:
Fixed eb027e76d81193d65b21e8dfb14c7b982cb3aae7.